code/game/objects/structures/watercloset.dm

```diff
@@ -295,6 +295,10 @@
 				if(H.belt.clean_blood())
 					H.update_inv_belt(0)
 			H.clean_blood(washshoes)
+
+		else if(istype(O, /mob/living/carbon/slime)) // Makes slimes to take damage from shower
```

Comment:
Hm, this would work, but imho it would be better to actually use the water reagent and call touch mob on that reagent.
This would also remove the need to extinguish the mob (224-227) since that's also handled by the water.

Reply:
Trying to make it work using water reagent